packages/tooling/fast-tooling-react/src/form/controls/utilities/control-switch.props.ts

```diff
@@ -1,3 +1,4 @@
+import { FormStrings } from "@microsoft/fast-tooling-react/src/form/form.props";
```

Comment:
Can this be a relative path (another one I spotted was below).

Reply:
Done